### PR TITLE
Move the cluster name of resources to annotations

### PR DESCRIPTION
**Pull request by calvin0327**
It's more appropriate to put the cluster message on annotions.

### CODE REVIEW

File: pkg/utils/clustername.go [hunk 1]
```diff
@@ -1,6 +1,7 @@
 package utils
 
 import (
+	"google.golang.org/genproto/googleapis/api/annotations"
```
Comment (Iceber) [inline pkg/utils/clustername.go:4]: Looks like unnecessary packages were imported.

Comment (calvin0327) [reply pkg/utils/clustername.go:4]: Let me clean up it.

**Comment by Iceber** [comment]
This modification will break the synchronized resources and the user needs to clean up the synchronized resource data.
> Currently in alpha stage, no need for compatibility processing
```sql
DROP TABLE resources;
```

**Review by Iceber — APPROVED** [review]
LGTM

**Pull request merged**